docs/api/math/Color.html

```diff
@@ -46,20 +46,20 @@
 
 		<h3>[name]( [page:Multi r], [page:Float g], [page:Float b] )</h3>
 		<div>
-		[page:Multi r] - (optional) the red component of the color if arguments g and b are defined.
-		If they are not defined, it can be a [link:https://en.wikipedia.org/wiki/Web_colors#Hex_triplet hexadecimal triplet] (recommended) or a CSS-style string or another Color instance.<br />
-		[page:Float g] - (optional) The green component of the color if it is defined.<br />
-		[page:Float b] - (optional) The blue component of the color if it is defined.<br /><br />
+		[page:Multi r] - (optional) If arguments [page:Float g] and [page:Float b] are defined, the red component of the color.
+		If they are not defined, it can be a [link:https://en.wikipedia.org/wiki/Web_colors#Hex_triplet hexadecimal triplet] (recommended), a CSS-style string, or another Color instance.<br />
+		[page:Float g] - (optional) If it is defined, the green component of the color.<br />
+		[page:Float b] - (optional) If it is defined, the blue component of the color.<br /><br />
 
 		Note that standard method of specifying color in three.js is with a [link:https://en.wikipedia.org/wiki/Web_colors#Hex_triplet hexadecimal triplet], and that method is used
 		throughout the rest of the documentation.<br /><br />
 
-		When all arguments are defined then r is the red component, g is the green component and b is the blue component of the color.<br />
-		When only r is defined:<br />
+		When all arguments are defined then [page:Multi r] is the red component, [page:Float g] is the green component and [page:Float b] is the blue component of the color.<br />
```

Comment:
If `Type` in `[page:Type name]` is a type that has an own  page in the docs, the link will lead to that page. But if that is not the case (if for instance `Type` is a Float or an Array), the type is only shown as a tooltip on mouseover.

Therefore I would suggest in this case, in which a direct linking makes no sense (since we have three possible types), to write something like `[page:Color_Hex_or_String r]`, so that the tooltip will display `Color_Hex_or_String`.